extensions/arc/src/test/mocks/fakeRadioButton.ts

```diff
@@ -0,0 +1,91 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the Source EULA. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import * as azdata from 'azdata';
+import * as vscode from 'vscode';
+
+export class FakeRadioButton implements azdata.RadioButtonComponent {
```

Comment:
we usually use 'Mock' instead 'Fake'

Reply:
There are already Fake AzdataApi and faceControllerModel objects here, do we need to change them all? Any particular reason to use one name above the other?